pkg/cloudprovider/cloud.go

```diff
@@ -167,3 +167,7 @@ func (s ServiceAnnotation) GetValue(key string) (string, bool) {
 	val, ok := s[key]
 	return val, ok
 }
+
+const (
+	LBAnnotationAllowSourceRange = "service.beta.kubernetes.io/load-balancer-source-ranges"
```

Comment:
Not sure why it leaves NodePorts in limbo.  Do you mean because we don't automatically configure them with a comparable annotation?  We don't currently configure any access for NodePorts.  So I think that is fine ... I see NodePorts as a building block e.g. for configuring a hardware load balancer, or for use for federated cloud interconnectivity.  I would imagine the component building on NodePorts would do whatever additional configuration it had to do, and so we should add extra support for NodePorts when we need it.

That said, I'm not sure exactly what motivated the addition of this annotation for GCE in the first place (I see the utility, just not sure whether there was something in particular because we didn't tie the original PR back to an issue)